src/cli/commands/pubsub/ls.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const utils = require('../../utils')
+const debug = require('debug')
+const log = debug('cli:pubsub')
+log.error = debug('cli:pubsub:error')
+
+module.exports = {
+  command: 'ls',
+
+  describe: 'Get your list of subscriptions',
+
+  builder: {},
+
+  handler (argv) {
+    utils.getIPFS((err, ipfs) => {
+      if (err) {
+        throw err
+      }
+
+      ipfs.pubsub.ls((err, subscriptions) => {
+        if (err) {
+          throw err
+        }
+
+        subscriptions.forEach((sub) => {
+          console.log(sub)
```

Comment:
Change to the `logger` thing @VictorBjelkholm added, so that we save time in refactoring.

https://github.com/ipfs/js-ipfs/issues/495